memflow-win32/src/win32/vat.rs

```diff
@@ -40,10 +40,15 @@ impl ScopedVirtualTranslate for Win32VirtualTranslate {
         addrs: VI,
         out: &mut VO,
         out_fail: &mut FO,
-        arena: &memflow::architecture::Bump,
+        tmp_buf: &mut [std::mem::MaybeUninit<u8>],
     ) {
-        let translator = x86::new_translator(self.dtb, self.sys_arch).unwrap();
-        translator.virt_to_phys_iter(mem, addrs, out, out_fail, arena)
+        if let Ok(translator) = x86::new_translator(self.dtb, self.sys_arch) {
```

Comment:
Cant we match on the architecture here before trying to create each translator just for clarity?